packages/webpack-cli/README.md

```diff
@@ -67,7 +67,6 @@ npx webpack-cli --help verbose
   help | h      Display help for commands and options
   version | v   Output version number of the 'webpack', 'webpack-cli' and other related packages
   init | c      Initialize a new webpack configuration
-  migrate | m   Migrate a configuration to a new version
```

Comment:
Keep it

Reply:
is appears when we register migrate command using `cli.makeCommand` in `migrate/index.ts` should I register it manually?

Reply:
No, if you have external package, it will be load automatically

Reply:
ah yes, updaing